services/QuillLMS/app/models/activity.rb

```diff
@@ -242,6 +254,7 @@ def connect_url_helper(initial_params)
   def module_url_helper(initial_params)
     return connect_url_helper(initial_params) if ['diagnostic', 'connect'].include?(classification.key)
     return lesson_url_helper if classification.key == 'lessons'
+    return comprehension_url_helper(initial_params) if classification.key == 'comprehension'
```

Comment:
I think we have this as a constant in the `ActivityClassification` model file.

Reply:
Good catch.  I've updated this line, as well as the two above it which I was copying the pattern from.